requirements.txt

```diff
@@ -0,0 +1,35 @@
+alembic==1.0.10
+botocore==1.12.172
+certifi==2019.6.16
+chardet==3.0.4
+Click==7.0
```

Comment:
why click? this is for command line apps or not?

Reply:
removed.